src/lib/meta.js

```diff
@@ -3,6 +3,8 @@ var escape = require('escape-html'),
     geojsonExtent = require('geojson-extent'),
     geojsonFlatten = require('geojson-flatten'),
     polyline = require('polyline'),
+    wkx = require('wkx'),
+    Buffer = require('buffer/').Buffer,
```

Comment:
You don't need to require Buffer - browserify will add the global when you refer to it.